src/input-mappings.js

```diff
@@ -100,14 +107,14 @@ const config = {
         s_up: "s_up",
         d_down: "d_down",
         d_up: "d_up",
-        W_down: "w_down",
-        W_up: "w_up",
-        A_down: "a_down",
-        A_up: "a_up",
-        S_down: "s_down",
-        S_up: "s_up",
-        D_down: "d_down",
-        D_up: "d_up"
+        arrowup_down: "w_down",
```

Comment:
Can we add the actual arrow keys here while we're at it?

Reply:
those are the actual arrow keys. This line is binding the up arrow press to the "w_down" action (which gets converted to movement by the `wasd-to-analog2d` component. It's a bit roundabout and should get better with changes to aframe-input-mappings